docs/infrastructure/deployment-targets/cloud-target-discovery/index.md

```diff
@@ -5,7 +5,17 @@ position: 100
 ---
 
 :::hint
-The Early Access Preview of the cloud target discovery feature was added in **Octopus 2022.1** and is enabled via **{{Configuration, Features}}**. The preview does not currently include removal of targets when the cloud resource is removed, this is on our roadmap.
+From **Octopus 2022.1**:
```

Comment:
I made some comments internally about this [here](https://octopusdeploy.slack.com/archives/C44HGJF8D/p1654764241824289?thread_ts=1654743111.083719&cid=C44HGJF8D) (private link)